Spring-Boot/Spring-Boot-Academy/src/test/java/tw/waterball/judgegirl/springboot/academy/controllers/HomeworkControllerTest.java

```diff
@@ -99,6 +102,27 @@ public void GivenOneHomeworkCreated_WhenGetHomeworkById_ShouldRespondHomework()
         assertEquals(homework, actualHomework);
     }
 
+    @Test
+    public void GivenOneHomeworkCreated_WhenDeleteHomeworkById_ShouldSucceed() throws Exception {
+        var homework = createHomeworkAndGet(HOMEWORK_NAME);
+
+        deleteHomework(homework.id).andExpect(status().isOk());
+
+        assertFalse(homeworkRepository.findHomeworkById(homework.id).isPresent());
```

Comment:
the human being is more familiar with the positive statement
so change it to

`assertTrue(homeworkRepository.findHomeworkById(homework.id).isEmpty());`